cmd/identities/get.go

```diff
@@ -29,10 +38,25 @@ func NewGetCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := cliclient.NewClient(cmd)
 
+			// we check includeCreds argument is valid
+			for _, opt := range includeCreds {
+				switch opt {
+				case "oidc":
+					break
+				default:
+					_, _ = fmt.Fprintln(os.Stderr, `You have to put a valid value of credentials type to be included, try --help for details.`)
+					os.Exit(1)
```

Comment:
Don't print to `os.Std...` but instead use `cmd.ErrOrStderr`. In tests you can then use a buffer and assert on the output.
Also, never `os.Exit` directly in the command, as it makes it impossible to test this. Always just return the error.

Reply:
Understood for `os.Exit`. 

I updated with `cmd.PrintErrln` which does both the print & `cmd.ErrOrStderr`

Reply:
Ah did not know `cmd.PrintErrln`, nice :+1: